celltype_annotation/utils/cellassign-helper-functions.R

```diff
@@ -0,0 +1,32 @@
+
+# define a function for obtaining cell type assignments from cell assign predictions
+get_celltype_assignments <- function(predictions){
+  
+  # get a dataframe of assignment for each barcode and prediction score
+  celltype_assignments <- predictions |>
+    tidyr::pivot_longer(!barcode,
+                        names_to = "celltype",
+                        values_to = "prediction") |>
+    dplyr::group_by(barcode) |>
+    dplyr::top_n(n = 1, wt = prediction) |>
```

Comment:
Apparently `top_n` is deprecated
```suggestion
    dplyr::slice_max(prediction, n = 1) |>
```